understanding/extended-audio-description-prerecorded.html

```diff
@@ -58,105 +56,90 @@
-                     <li><a href="#brief">In Brief</a></li>
-                     <li><a href="#intent">Intent</a></li>
-                     <li><a href="#benefits">Benefits</a></li>
-                     <li><a href="#examples">Examples</a></li>
-                     <li><a href="#resources">Related Resources</a></li>
-                     <li><a href="#techniques">Techniques</a></li>
-                     <li><a href="#key-terms">Key Terms</a></li>
+                     <li><a href="#brief">要約</a></li>
+                     <li><a href="#intent">意図</a></li>
+                     <li><a href="#benefits">利点</a></li>
+                     <li><a href="#examples">事例</a></li>
+                     <li><a href="#resources">関連リソース</a></li>
+                     <li><a href="#techniques">テクニック</a></li>
+                     <li><a href="#key-terms">重要な用語</a></li>
                   </ul>
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				SC 1.2.7:</span>Extended Audio Description (Prerecorded) (Level AAA)
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				達成基準 1.2.7:</span>拡張音声解説 (収録済) (レベル AAA)
+
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
-                  <p>Where pauses in foreground audio are insufficient to allow <a href="#dfn-audio-description">audio descriptions</a> to convey the sense of the video, <a href="#dfn-extended-audio-description">extended audio description</a> is provided for all <a href="#dfn-prerecorded">prerecorded</a>
-                     <a href="#dfn-video">video</a> content in <a href="#dfn-synchronized-media">synchronized media</a>.
+                  <p>前景音の合間が、<a href="#dfn-audio-description">音声解説</a>で映像の意味を伝達するのに不十分な場合、<a href="#dfn-synchronized-media">同期したメディア</a>に含まれているすべての<a href="#dfn-prerecorded">収録済</a>の<a href="#dfn-video">映像</a>コンテンツに対して、<a href="#dfn-extended-audio-description">拡張音声解説</a>が提供されている。
                      
                   </p>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Videos can be played with more detailed audio descriptions.</dd>
+                  <dt>目標</dt>
+                  <dd>映像を、より詳細な音声解説付きで再生できるようにする。</dd>
```

Comment:
「映像を」にするのであれば、語順を入れ替えてしまうのがよいと思います。
```suggestion
                  <dd>より詳細な音声解説付きで映像を再生できるようにする。</dd>
```